feature-libs/checkout/base/core/connectors/checkout-billing-address/checkout-billing-address.adapter.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Address } from '@spartacus/core';
+import { Observable } from 'rxjs';
+
+export abstract class CheckoutBillingAddressAdapter {
+  /**
+   * Abstract method used to set address for billing
+   *
+   * @param userId
+   * @param cartId
+   * @param addressId
+   */
+  abstract setAddress(
```

Comment:
What about change name to 'setBillingAddress'?

Reply:
changed